dist/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>CitizenNet UI Guide</title>
+    <base href="/lynx/" />
```

Comment:
Looks like this breaks the page unless we're serving this directory as `/lynx`. Can we drop this in favor of fixing the ingress to point to `/lynx/`? That should make this page work both in our k8s workflow and for normal HTML stuff.